weave

```diff
@@ -1667,7 +1667,31 @@ launch_router() {
            --label=weavevolumes $WEAVEDB_IMAGE >/dev/null
     fi
 
-    RESOLV_CONF=$(chroot ${HOST_ROOT:-/} readlink -f /etc/resolv.conf)
+    echo "Container PATH is $PATH"
+    # Avoid readlink if not present in FHS directories of host
+    if test -x ${HOST_ROOT:-/}/bin/readlink \
```

Comment:
so perhaps simply
1. run `test -L ...` to determine whether we are looking at a symlink
2. if false then no-op
3. if true then attempt the chroot'ed readlink

This doesn't give us a nice error when readlink isn't there but IMO the stock error is quite ok.

Or perhaps even just
```
RESOLV_CONF=/etc/resolv.conf
[ -f ${HOST_ROOT:-/}$RESOLV_CONF ] || RESOLV_CONF=$(chroot ${HOST_ROOT:-/} readlink -f $RESOLV_CONF)
```